File: debot-no-op/src/main/java/com/tomoima/debot/Debot.java
package com.tomoima.debot;

import android.content.Context;
import android.support.v4.app.FragmentActivity;

public class Debot {

    private Debot(){
        //Do nothing
    }

    public static Debot getInstance(){
        return new Debot();
    }


    public void allowShake(Context context) {
    }

    public void startSensor(FragmentActivity activity) {
    }

    public void stopSensor() {
    }

    public void showDebugMenu(FragmentActivity activity) {
    }

}

File: debot-no-op/src/main/java/com/tomoima/debot/Debot.kt
package com.tomoima.debot

import android.content.Context
import androidx.fragment.app.FragmentActivity

class Debot private constructor()//Do nothing
{

    fun allowShake(context: Context) {}

    @JvmOverloads
    fun startSensor(activity: FragmentActivity, sensorDelay: Int = 0) {}

    fun stopSensor() {}

    fun showDebugMenu(activity: FragmentActivity?) {}

    companion object {

        @JvmStatic fun getInstance(): Debot = Debot()
    }

}
